terraform/deployments/govuk-publishing-platform/dns.tf

```diff
@@ -57,5 +59,5 @@ resource "aws_route53_record" "workspace_public" {
 
 resource "aws_acm_certificate_validation" "workspace_public" {
   certificate_arn         = aws_acm_certificate.workspace_public.arn
-  validation_record_fqdns = [for record in aws_route53_record.workspace_public : record.fqdn]
+  validation_record_fqdns = [for record in aws_route53_record.workspace_public : record.name]
```

Comment:
Is this right? `validation_record_fqdns` makes it sound like it wants FQDNs, not subdomains? Am I misunderstanding?

Reply:
In theory you are right it should be done this way, the issue is we don't control `*.publishing.service.gov.uk` domain in this repo, which makes it tedious to get the zone_id to include in `aws_route53_record.workspace_public`. 

This is the workaround I found.